dvclive/lightning.py

```diff
@@ -0,0 +1,80 @@
+from typing import Dict, Optional, Union
+
+from pytorch_lightning.loggers import LightningLoggerBase
+from pytorch_lightning.loggers.base import rank_zero_experiment
+from pytorch_lightning.utilities import rank_zero_only
+from torch import is_tensor
+
+import dvclive
+
+
+class DvcLiveLogger(LightningLoggerBase):
+    def __init__(
+        self,
+        run_name: Union[str, int],
+        save_dir: Optional[str] = None,
+        summary: Optional[bool] = True,
+        prefix="",
+        experiment=None,
+    ):
+
+        if dvclive is None:
+            raise ImportError(
+                "You want to use `dvclive` logger which is not installed yet,"
+                " install it with `pip install dvclive`."
+            )
+
+        super().__init__()
+        self._prefix = prefix
+        self._dvclive_init = dict(
+            path=save_dir,
+            summary=summary,
+        )
+        self._experiment = experiment
+        self._version = run_name
+
+    @property
+    def name(self):
+        return "DvcLiveLogger"
+
+    @rank_zero_only
+    def log_hyperparams(self, params, *args, **kwargs):
+        pass
+
+    @property
+    @rank_zero_experiment
+    def experiment(self):
+        r"""
+        Actual DVCLive object. To use DVCLive features in your
+        :class:`~LightningModule` do the following.
+        Example::
+            self.logger.experiment.some_dvclive_function()
+        """
+        if self._experiment is not None:
+            return self._experiment
+        else:
+            assert (
+                rank_zero_only.rank == 0
+            ), "tried to init log dirs in non global_rank=0"
+            self._experiment = dvclive.init(**self._dvclive_init)
+
+        return self._experiment
+
+    @property
+    def version(self):
+        return self._version
+
+    @rank_zero_only
+    def log_metrics(
+        self, metrics: Dict[str, float], step: Optional[int] = None
+    ):
+        assert (
+            rank_zero_only.rank == 0
+        ), "experiment tried to log from global_rank != 0"
```

Comment:
Do we really need this assertions if we are using the `@rank_zero_only` decorator?

```suggestion
```

Reply:
I've seen this line in several Lightning loggers. For example: [wandb logger](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/loggers/wandb.py#L212), [comet logger](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/loggers/comet.py#L241), [neptune logger](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/loggers/neptune.py#L469)
I assume that unexpected situations may occur on distributed GPUs, so I left this assertion just in case

Reply:
Got it. Thanks!